content/release-10-18-0/devices-10-18-0.md

```diff
@@ -94,6 +108,35 @@ Fix Version</th>
 <td>10.17.16.0</td>
 </tr>
 
+<tr>
+<td>Loriot</td>
+<td>Events created by the Loriot agent when sending uplink requests are now considered as device requests and provide connectivity status and availability monitoring.</td>
+<td>DM-2139</td>
+<td>10.17.170.0</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>During the LWM2M client registration, if the agent failed to find the registered device from the database due to a communication problem, an internal object related to the device managed object was automatically removed by the agent. Afterwards, LWM2M clients always failed to connect. This is now fixed and the internal object is only removed if the actual device managed object does not exist in the database.</td>
```

Comment:
```suggestion
<td>During the LWM2M client registration, if the agent failed to find the registered device in the database due to a communication problem, an internal object related to the device managed object was automatically removed by the agent. Afterwards, LWM2M clients always failed to connect. This is now fixed and the internal object is only removed if the actual device managed object does not exist in the database.</td>
```

we had this entry before, not sure about the preposition change.

Reply:
I also considered this but I´m not sure if it is "the device from the database" or "find the device in the database" which is why I left it as is.